ts/input/tex/StackItem.ts

```diff
@@ -28,7 +28,7 @@ import TexError from './TexError.js';
 import StackItemFactory from './StackItemFactory.js';
 
 // Union types for abbreviation.
-export type EnvProp = string | number | boolean;
+export type EnvProp = string | number | boolean | RegExp;
```

Comment:
I don't like that. I'd like to keep that to primitive types.

Reply:
Other than aesthetic reasons, is there a practical advantage to that?  I'm not sure why the environment properties can't be any type of data that is needed.